Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_016.StripeApiKey.sarif

```diff
@@ -44,7 +44,7 @@
           "message": {
             "id": "Default",
             "arguments": [
-              "SEC101_016.StripeApiKey.txt",
+              "sk_tes…",
```

Comment:
>sk_tes [](http://example.com/codeflow?start=15&length=6)

note that some secret kinds have prefixes that already consume all the available truncation space without providing any per-secret data. We could consider allowing rules to provide a different truncation value in the future (longer than the default of 6 chars).